.github/workflows/build-ubuntu.yml

```diff
@@ -0,0 +1,55 @@
+name: Build on Ubuntu
+
+on:
+  push:
+    branches:
+      - master
+      - v*
+
+    tags:
+      - v*
+
+    paths:
+      - cmake/**
+      - src/**
+      - CMakeLists.txt
```

Comment:
What is being skipped here? Just let it run always...

Reply:
Uh, `data/**` ? A lot of commits for datapack only. This artifact doesn't include datapack here.

Reply:
Ah, that makes sense, I didn't remember data :P it could be a separate repo as a submodule.